index.html

```diff
@@ -8,15 +8,7 @@
 </head>
 <body>
     <h1 id="title">Michael Tribute Page</h1>
-
-    <figure id="img-div">
```

Comment:
Removing unused elements is cool and really not a big deal! Maybe try to find a better use for them in the future! - Optional Suggestion